### PR TITLE
Revisit client API docs

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: docs/api/client/http-api.md [hunk 1]
```diff
@@ -143,3 +143,20 @@ const queryViewModel = async (viewModelName, aggregateIds, jwt) => {
...
+  ...
+  headers: {
+    Authorization: `Bearer ${jwt}`,
+    ...
+  },
+  ...
+})
+```
+
+See the [Authentication and Authorization](../../authentication-and-authorization.md) topic for information on how authorization is handled on server.
```
Comment (albertov05) [inline docs/api/client/http-api.md:162]:
```suggestion
See the [Authentication and Authorization](../../authentication-and-authorization.md) topic for information on how authorization is handled on the server.
```

File: docs/api/client/resolve-client.md [hunk 2]
```diff
@@ -74,17 +117,39 @@ var imagePath = client.getStaticAssetUrl('/account/image.jpg')
...
 
-##### Example
+| Argument Name       | Description                                                                     |
+| ------------------- | ------------------------------------------------------------------------------- |
+| url                 | A URL used to establish a WebSocket connection to a view model.                 |
+| cursor              | The data cursor used to traverse the events included into the query result set. |
+| viewModelName       | A string that specifies the name of a view model.                               |
+| aggregateIds        | A list of aggregate IDs for which to receive events.                            |
+| handler             | A function that handles incoming events.                                        |
+| subscribeCallback   | A callback called on successful subscription or error.                          |
```
Comment (albertov05) [inline docs/api/client/resolve-client.md:147]:
```suggestion
| subscribeCallback   | A callback called on a successful subscription or an error.                          |
```

File: docs/api/client/resolve-client.md [hunk 3]
```diff
@@ -74,17 +117,39 @@ var imagePath = client.getStaticAssetUrl('/account/image.jpg')
...
+| resubscribeCallback | A callback called on successful resubscription or error.                        |
```
Comment (albertov05) [inline docs/api/client/resolve-client.md:148]:
```suggestion
| resubscribeCallback | A callback called on a successful resubscription or an error.                        |
```

File: docs/api/client/resolve-redux.md [hunk 4]
```diff
@@ -8,17 +8,17 @@ The reSolve framework includes the client **@resolve-js/redux** library used to
...
-| ------------------------------------------------------- | --------------------------------------------------------------------------- |
-| [useReduxCommand](#usereduxcommand)                     | Creates a hook to execute a command.                                        |
-| [useReduxReadModel](#usereduxreadmodel)                 | Creates a hook to query a Read Model.                                       |
-| [useReduxReadModelSelector](#usereduxreadmodelselector) | Creates a hook to access a Read Model query result.                         |
-| [useReduxViewModel](#usereduxviewmodel)                 | Creates a hook to receive a View Model's state updates and reactive events. |
-| [useReduxViewModelSelector](#usereduxviewmodelselector) | Creates a hook to access a View Model's current state on the client.        |
+| Function Name                                           | Description                                             |
+| ------------------------------------------------------- | ------------------------------------------------------- |
+| [useReduxCommand](#usereduxcommand)                     | Creates a hook that execute a command.                  |
```
Comment (albertov05) [inline docs/api/client/resolve-redux.md:13]:
```suggestion
| [useReduxCommand](#usereduxcommand)                     | Creates a hook that executes a command.                  |
```

File: docs/api/client/resolve-redux.md [hunk 5]
```diff
@@ -8,17 +8,17 @@ The reSolve framework includes the client **@resolve-js/redux** library used to
...
+| [useReduxReadModel](#usereduxreadmodel)                 | Creates a hook that queries a Read Model.               |
+| [useReduxReadModelSelector](#usereduxreadmodelselector) | Creates a hook used access a Read Model query result.   |
```
Comment (albertov05) [inline docs/api/client/resolve-redux.md:15]:
```suggestion
| [useReduxReadModelSelector](#usereduxreadmodelselector) | Creates a hook used to access a Read Model query result.   |
```

File: docs/api/client/resolve-redux.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ const { execute: toggleItem } = useReduxCommand({
 
 ### useReduxReadModel
 
-Creates a hook to query a reSolve Read Model
+Creates a hook that queries a Read Model,
```
Comment (albertov05) [inline docs/api/client/resolve-redux.md:38]:
```suggestion
Creates a hook that queries a Read Model.
```

File: docs/frontend.md [hunk 7]
```diff
@@ -302,13 +278,16 @@ The [shopping-list-redux-hoc](https://github.com/reimagined/resolve/tree/master/
...
-| [useQuery](api/client/resolve-react-hooks.md#usequery)                   | Allows a component to send queries to a reSolve Read Model or View Model. |
-| [useOriginResolver](api/client/resolve-react-hooks.md#useoriginresolver) | Resolves a relative path to an absolute URL within the application.       |
+| Hook                                                                     | Description                                                                               |
+| ------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------- |
+| [useСlient](api/client/resolve-react-hooks.md#useclient)                 | Returns the [@resolve-js/client](api/client/resolve-client.md) library's `client` object. |
+| [useCommand](api/client/resolve-react-hooks.md#usecommand)               | Initializes a command that can be passed to the backend.                                  |
+| [useCommandBuilder](api/client/resolve-react-hooks.md#usecommandbuilder) | Allows to generate commands based on input parameters.                                    |
```
Comment (albertov05) [inline docs/frontend.md:285]: _Who_ is allowed to generate commands? Please specify.

Comment (EugeniyBurmistrov) [reply docs/frontend.md:285]: Changed to "allows  a component to generate commans"

Comment (albertov05) [reply docs/frontend.md]: Thank you!

**Review by max-vasin — APPROVED** [review]

**Review by const314 — APPROVED** [review]

**Pull request merged**